Parking/ParkingSpot/doc/spec.md

```diff
@@ -78,7 +79,8 @@ only one group is allowed per parking spot.
         + Other values as per application needs
     + Mandatory
     
-+ `TimeInstant` : Timestamp saved by FIWARE's IoT Agent. Note: This attribute has not been harmonized
++ `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant)
```

Comment:
Just confirming with you that you really want to link it to the `develop` branch of the https://github.com/telefonicaid/iotagent-node-lib project instead of `master` (where the stable, not the development version is) :)